src/playbin.js

```diff
@@ -87,6 +87,15 @@ var Playbin = GObject.registerClass({
         this._video_widget.show();
         this.add(this._video_widget);
 
+        this._has_ready_to_show = !!GObject.signal_lookup('ready-to-show',
+            this._video_widget.constructor.$gtype);
```

Comment:
Does  "!!" do something special? 
Isn't this equivalent to
```
        this._has_ready_to_show = GObject.signal_lookup('ready-to-show',
            this._video_widget.constructor.$gtype);
```

Reply:
`!!` converts to a boolean. It's almost equivalent, but having a boolean is slightly easier when debugging because you don't have to wonder what `this._has_ready_to_show === 379` means.

Reply:
Then it's okay as it is.